tfjs-backend-webgpu/src/kernels/FromPixels.ts

```diff
@@ -24,6 +24,35 @@ import * as webgpu_program from './webgpu_program';
 import {FromPixelsProgram} from './FromPixels_utils/from_pixels_webgpu';
 import {util} from '@tensorflow/tfjs-core';
 
+var uint8ToFloat16Table = [
```

Comment:
Can you add a comment above the line to explain how to get this table? What's your reference or algorithm?

Reply:
Done